__tests__/app.test.js

```diff
@@ -52,4 +52,57 @@ describe("Get /api", () => {
         
         })
     })
-})
\ No newline at end of file
+})
+
+describe("Get /api/articles/:article_id", () => {
+    test("200: responds with the first article with the relevant properties when passed an article_id of 1", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toBeInstanceOf(Object)
+            expect(body.article).toHaveProperty("author")
+            expect(body.article).toHaveProperty("title")
+            expect(body.article).toHaveProperty("topic")
+            expect(body.article).toHaveProperty("article_id")
```

Comment:
We want to assert that we expect article id 1 here, otherwise we can't be confident that we are getting the correct item back looking at this test